### PR TITLE
Doc: Fix broken links

### PR DESCRIPTION
**Pull request by lianhao**
## Description

Fix broken links.

## Issues

`n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Dependencies

n/a

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Review by zhlsunshine — APPROVED** [review]
LGTM

**Review by Ruoyu-y — APPROVED** [review]
LGTM

File: microservices-connector/config/samples/ChatQnA/use_cases.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ For Gaudi:
 - tgi-service: ghcr.io/huggingface/tgi-gaudi:1.2.1
 
 > [NOTE]  
-> Refer to [Xeon README](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/xeon/README.md) or [Gaudi README](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/README.md) to build the OPEA images. These too will be available on Docker Hub soon to simplify use.
+> Refer to [Xeon README](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker_compose/intel/cpu/xeon/README.md) or [Gaudi README](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker_compose/intel/hpu/gaudi/README.md) to build the OPEA images. These too will be available on Docker Hub soon to simplify use.
```
Comment (Ruoyu-y) [inline microservices-connector/config/samples/ChatQnA/use_cases.md:32]: One typo. These "two" instead of "too"

Comment (lianhao) [reply microservices-connector/config/samples/ChatQnA/use_cases.md:32]: I'll remove this NOTE section, since these images are already on docker hub

**Pull request merged**